docs/using/alerting.md

```diff
@@ -136,46 +124,62 @@ For example, to validate that a MongoDB instance is down, shut down a member of
 {service_type="mongodb"}
 ```
 
-## Template compatibility
-### With previous PMM versions
-PMM 2.26 introduced significant changes to the core structure of rule templates. As a result, alert rules and templates created in previous PMM versions are incompatible with PMM 2.26.0 and cannot be migrated to this new version.
+### Add an alert rule
+After provisioning the resources required for creating Percona templated alerts, you are now ready to create your alert:
 
-If you’re migrating from PMM 2.25 and earlier, make sure to manually recreate any custom alert rules and rule templates that you want to transfer to PMM 2.26.0 or later.
+1. Go to **Alerting > Alert Rules**, and click **New alert rule**.
+2. On the **Create alert rule** page, select the **Percona templated alert** option. If you want to learn about creating Grafana alerts instead, check our [Grafana's documentation](https://grafana.com/docs/grafana/latest/alerting/).
+3. In the **Template details** section, choose the template on which you want to base the new alert rule. This automatically populates the **Name**, **Duration**, and **Severity** fields with information from the template. You can change these values if you want to override the default specifications in the template.
+4. In the **Filters** field, specify if you want the alert rule to apply only to specific services or nodes. For example: `service_name'`, Operator:`=(EQUAL)`, VALUE: `ps5.7`.
```

Comment:
```suggestion
4. In the **Filters** field, specify if you want the alert rule to apply only to specific services or nodes. For example: `service_name'`, Operator:`MATCH`, VALUE: `ps5.7`.
```

Reply:
That part was changed, now we have two filters types: `MATCH` and `MISMATCH`, both works as a RegEx.